react/fields/Dropdown/Dropdown.js

```diff
@@ -178,7 +137,7 @@ export default class Dropdown extends Component {
 
     return (
       <div className={classNames}>
-        {this.renderLabel()}
+        <FieldLabel {...this.props} />
```

Comment:
Sure, the validation I understand living in the component that cares. I'm not sure about blindly passing everything though. @markdalgleish you have any thoughts on this?